### PR TITLE
Update support libraries to be able to parse appinfo.vdf v29 format

### LINKED ISSUE #14: "Invalid magic" when scanning Steam library
For the last week or two, I haven't had any icons showing in Keypirinha when I was launching a game, and today I did a little digging.

I get this in console when refreshing the catalogue

```
17:58:08.301 ** ERROR: Error from Steam.Steam.on_catalog: <class 'SyntaxError'>: Invalid magic, got b')DV\x07'
Traceback (most recent call last):
  File "<REMOVED>\Steam.keypirinha-package\steam.py", line 64, in on_catalog
  File "<REMOVED>\Steam.keypirinha-package\steam.py", line 143, in get_applist
  File "<REMOVED>\Steam.keypirinha-package\lib\appcache.py", line 73, in parse_appinfo
```

I think there's been an issue like this before, when Steam updated their file format, maybe?

### PR DESCRIPTION
**Pull request by FlyveHest**
Updated appinfo parser and support libraries from https://github.com/solsticegamestudios/steam

Closes #14

### CODE REVIEW

File: keypirinha-steam/src/steam.py [hunk 1]
```diff
@@ -140,7 +140,7 @@ def get_applist(self, steam_dir):
 
         # Load appinfo.vdf to extract info about games
         with open(appinfo_path, 'rb') as fp:
-            _, steamapps = appcache.parse_appinfo(fp)
+            _, steamapps = appcache.parse_appinfo(fp, dict)
```
Comment (EhsanKia) [inline keypirinha-steam/src/steam.py:143]: is this needed? I think the default for mapper is `dict` down the line anyways?

File: keypirinha-steam/README.md [hunk 2]
```diff
@@ -44,3 +48,4 @@ If needed, you can use the Keypirinha console to debug your Launchy configuratio
 - 2.2: Fix bug when a game doesn't have an icon
 - 2.3: Add support for Steam applications
 - 2.4: Fixed issue with Integer game names
+- 2.6: Updated various libraries top be able to parse appinfo.vdf v29 format
```
Comment (EhsanKia) [inline keypirinha-steam/README.md:52]: nit: I technically forgot to update this, can you add

- 2.5: Update appinfo parsing library to support v28 format

**Comment by EhsanKia** [comment]
This is awesome, thank you so much :)

File: keypirinha-steam/README.md (hunk 2 above)
Comment (FlyveHest) [reply keypirinha-steam/README.md:52]: Added and pushed 👍

File: keypirinha-steam/src/steam.py (hunk 1 above)
Comment (FlyveHest) [reply keypirinha-steam/src/steam.py:143]: I thought so as well, but in my testing it didn't seem to work without specifying dict (got an error about an object not being a subclass)

**Pull request merged**